src/Bootstrappers/BatchTenancyBootstrapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Stancl\Tenancy\Bootstrappers;
+
+use Illuminate\Bus\BatchRepository;
+use Illuminate\Bus\DatabaseBatchRepository;
+use Illuminate\Support\Facades\DB;
+use ReflectionClass;
+use Stancl\Tenancy\Contracts\TenancyBootstrapper;
+use Stancl\Tenancy\Contracts\Tenant;
+
+class BatchTenancyBootstrapper implements TenancyBootstrapper
+{
+    private $previousConnection = null;
```

Comment:
Add docblock, type, and change this to `protected`

Reply:
Added.